js/gal.js

```diff
@@ -88,6 +106,34 @@ outcodeRandomPostcodeButton.addEventListener('click', () => {
         .catch(handleError);     
 });
 
+// Unique Postcode Section 
+
+async function uniquePostcodeGenerator (postcode) {
+        // Variables
+        const response = await fetch(`${uniquePostcodeEndpoint}/${postcode}`);
+        const data = await response.json();
+        const latitude = data.result.latitude;
+        const longitude = data.result.longitude;
+        console.log(data);
+
+        localPoliceForceGenerator(latitude, longitude)
+        .catch(handleError);
+
+}
+
+// Data Police API Section 
+
+async function localPoliceForceGenerator (latitude, longitude) {
+        // Variables
+        const response = await fetch(`${localPoliceForceEndpoint}?q=${latitude},${longitude}`);
+        const data = await response.json();
+        console.log(data);
+}
+
+
+uniquePostcodeGenerator('nr279fd')
```

Comment:
How come you're running these functions here? Shouldn't they only run when a user has submitted a form. When I'm running your site on my local machine, both the inputs are already filled in and I think it might be related to these inputs. Is that really what you want to happen?

Reply:
This is just my testing currently. They won't be present at shipping 🙂